scss/button/_layout.scss

```diff
@@ -118,9 +118,21 @@
     }
 
     //DropDown Button
-    kendo-dropdownbutton.k-state-focused,
-    kendo-dropdownbutton:focus {
-        outline: 0;
+    .k-dropdownbutton {
```

Comment:
If we have `k-split-button`, then this should be `k-dropdown-button`, or vice versa -- `k-splitbutton` and `k-dropdownbutton`.

The rest is fine.